readme.md

```diff
@@ -161,12 +161,17 @@ Enable it in your `package.json` with the `extends` option:
 {
 	"name": "my-awesome-project",
 	"eslintConfig": {
-		"extends": "plugin:unicorn/recommended"
+		"plugins": [
+			"unicorn"
```

Comment:
The recommended config already includes this.

Reply:
Hmmm, I thought that is missing, I should close this.